examples/heat_equation.cpp

```diff
@@ -254,11 +265,12 @@ int main(int argc, char** argv)
     display(ddc::coordinate(time_domain.front()),
             ghosted_temp[x_domain][y_domain]);
     // time of the iteration where the last output happened
-    ddc::DiscreteElement<DDimT> last_output = time_domain.front();
+    ddc::DiscreteElement<DDimT> index_time_last_output
+            = time_domain.front();
     //! [initial output]
 
     //! [time iteration]
-    for (auto const iter :
+    for (auto const idxDDimT :
```

Comment:
What do you think of `time_delem` instead of `idxDDimT` ?

Reply:
The `delem` suffix could be confusing. I tried to follow the naming conventions described in Emily`s tutorial. Or maybe just `time_idx`? That should be clear for everyone.

Reply:
`time_idx` would be fine to me but would not match the current convention. May I ask why do you think `delem` is confusing ?

Reply:
If I'm correct, you're referring to Gyselalib++ naming convention tutorial. A different naming convention is used in DDC

Reply:
I was at first confused for a short time what `delem` is an abbreviation for, but it is `DiscreteElement`, right? I just got confused because I think of `double` when I see a `d` due to Gysela. But yes sorry I forgot that DDC does not follow Gysela convention, of course. Where can I learn the DDC naming convention i.e. where can I find it? In the other examples `ix` is used for a `ddc::DiscreteElement` in x-direction e.g. Is this the DDC convention?

Reply:
I don't think you need to bother learning DDC conventions, they are more for DDC developpers.

About `ix` you are right, we are currently not consistent with our new convention.

In the end, if we change `iter` there will still be inconsistencies with other discrete elements, what do you think we keep it this way and open a new issue to rename them all at once ?

Reply:
That sounds great. Then I will leave `iter` as it was.